targets/stm32f405.ld

```diff
@@ -0,0 +1,9 @@
+MEMORY
+{
+    FLASH_TEXT (rw) : ORIGIN = 0x08000000, LENGTH = 1M
+    RAM (xrw)       : ORIGIN = 0x20000000, LENGTH = 128K
+}
+
+_stack_size = 4K;
+
+INCLUDE "targets/arm.ld"
```

Comment:
It's exactly the same as stm32f407.ld, so I think it's better to unify it.
Any reason why it would be better to separate them?

Reply:
The original script I used had quite a bit more configuration, but I replaced it with the `stm32f407.ld` for simplicity - to define only the bare minimum. I would personally prefer keeping it there for now and "upgrading" it after (if) the board gets merged

Reply:
ok.